content/en/tracing/profiler/enabling/node.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Enabling the Node.js Profiler
+kind: Documentation
+code_lang: node
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/search_profiles'
+      tag: 'Documentation'
+      text: 'Learn more about available profile types'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
```

Comment:
Can you add https://github.com/DataDog/documentation/blame/master/content/en/tracing/profiler/enabling/ruby.md#L19-L23?